src/lib/ui/components/Select/Select.svelte

```diff
@@ -89,6 +90,13 @@
     }
   }
 
+  $: onHide = active
+    ? () => {
+        handleToggle();
+        setField(name, $data[name] ?? "");
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/indent](https://typescript-eslint.io/rules/indent)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 6 spaces but found 8.
```suggestion
      setField(name, $data[name] ?? "");
```